README.md

```diff
@@ -5,167 +5,145 @@
 [![Build Status][1]][2]
 
 ## Overview
-Heptio Ark is a utility for managing disaster recovery, specifically for your [Kubernetes][14] cluster resources and persistent volumes. It provides a simple, configurable, and operationally robust way to back up and restore applications and PVs from a series of checkpoints. This allows you to better automate in the following scenarios:
 
-* **Disaster recovery** with reduced TTR (time to respond), in the case of:
-    * Infrastructure loss
-    * Data corruption
-    * Service outages
+Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
 
-* **Cross-cloud-provider migration** for Kubernetes API objects (cross-cloud-provider migration of persistent volume snapshots not yet supported)
+* Take backups of your cluster and restore in case of loss.
+* Copy cluster resources across cloud providers. NOTE: Cloud volume migrations are not yet supported.
+* Replicate your production environment for dev and testing environments.
 
-* **Dev and testing environment setup (+ CI)**, via replication of prod environment
+Ark consists of:
 
-More concretely, Heptio Ark combines an in-cluster service with a CLI that allows you to record both:
-1. *Configurable subsets of Kubernetes API objects* -- as tarballs stored in object storage
-2. *Disk snapshots of Persistent Volumes* -- via the cloud provider APIs
+* A server that runs on your cluster
+* A CLI
```

Comment:
Perhaps "A command line client that runs on a local computer?" The server part mentions where it runs, but the line for the CLI could be a bit ambiguous for some.